### ISSUE TITLE
Add EPUB3 to the list of documents

### ISSUE THREAD

**Issue by mgifford** [wet-boew/wet-boew#8529]
There's a list of approved document types here:
  http://wet-boew.github.io/wet-boew-styleguide/v4/writing/strctr-en.html

This page supports the following file types HTML, Word, PDF, PPT, Excel but doesn't mention EPUB.

EPUB is a better tool to convey information in a mobile first era than anything other than HTML. It many cases it is better than HTML. For instance, many of the long directives that governments tend to produce are hard to read.  Having something that is easier to page through on a phone would be easier. 

PDFs are best for something you would want to print out.  If you don't want to print it out but want a portable document, the default should be EPUB. 

For many Canadians still struggling with reliable internet, being able to easily download a portable document to their phone is key to accessing government services.

**Comment by shawnthompson** [comment]
I believe this documentation was brought in from CRA's internet style guide. I think it talks about HTML being the best way to post information online and discourages people from posting any other format.

 > Publication of online content in formats other than HTML, such as Word, PDF, PPT, or Excel, goes against most usability principles 

Curious if [the GC digital operations strategic plan (2018-2022)](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html) will replace this outdated information on WET? 

#TL;DR

**Comment by mgifford** [comment]
Interesting.. The CRA has some good information on their Intranet site on accessibility.  I do think it would be useful to find a way to share some of this more broadly.  

One of the advantages of sharing is that it helps provide a space to discuss/debate if this is still a best practice.  I'd love to see more acceptance of EPUB in government as it addresses needs for a broader selection of users than PDFs.  HTML is always good of course, but it has it's own limitations.

**Comment by shawnthompson** [comment]
I know @jrowsell was looking into ePub and she's now with TBS so she might be in a better position to get the talk started over there.

**Comment by jvanulde** [comment]
EPUB by its very nature is exclusionary. We should adopt HTML as the standard for all documents and scientific papers in the GC. This is the only path forward if we are serious about ML and AI.

**Comment by shawnthompson** [comment]
But what about those who want to publish to PDF (accessible) instead of HTML? I think we shoulod replacing PDFs with ePubs instead. I use VoiceOver to read long documents online, ePubs work great in iOS but PDF, even accessible ones are tricky.

**Comment by shawnthompson** [comment]
Can you make a responsive PDF? I was told you could but never seen it done.

**Comment by mgifford** [comment]
I do think it is or will be possible.  However, it will be a decade or more before most PDF files are responsive.  The challenge isn't with the format itself, as much as the authoring tools and author practices that produce the ocean of PDF files.  The vast majority of which aren't accessible or responsive.  There's no way of telling which were developed using 2018 best practices and which were just exported from Photoshop.

**Comment by jvanulde** [comment]
You need a plugin to view PDF. While those plugins are pervasive you need to ask yourself why they are needed in the first place. We need to get out of the headspace where we treat the web like a repository. Instead, we need to encode data and information into the web so that it can be crawled and reasoned over using agents (e.g. ML, AI). We need to drop this "publishing" paradigm that is a throwback to the days where information was stored on scrolls. We need to look forward.

**Comment by mgifford** [comment]
PDFs mostly open by default in the browser.  They don't always for EPUB, although I think IE does now.

There's an issue for it in FF
https://bugzilla.mozilla.org/show_bug.cgi?id=770946

**Comment by jvanulde** [comment]
@mgifford my point exactly.

**Comment by mgifford** [comment]
On EPUB3 here's an accessibility validator that leverages aXe-Core https://github.com/daisy/ace

Some great work done here https://inclusivepublishing.org & by the Daisy folks.

It's worth asking too where the automated tools are for evaluating PDF files are.  I haven't found any good ones that compare with what is available in HTML.
